runbot_render_logs/__openerp__.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+{
+    'name': "runbot_render_logs",
+
+    'summary': """
+        Short (1 phrase/line) summary of the module's purpose, used as
+        subtitle on modules listing or apps.openerp.com""",
+
+    'description': """
+        Long description of module's purpose
+    """,
+
+    'author': "My Company",
+    'website': "http://www.yourcompany.com",
+
+    'category': 'Uncategorized',
+    'version': '0.1',
+
+    # any module necessary for this one to work correctly
+    'depends': ['base', 'runbot', 'runbot_travis2docker'],
```

Comment:
@moylop260 runbot_travis2docker is necessary because if I remove it, does not pass through the overwriting of job_10_test_base method